webapp/templates/rewriterule-create.xhtml

```diff
@@ -0,0 +1,176 @@
+                    <thead>
+                        <tr>
+                            <th class="col-sm-5" colspan="2">Request</th>
+                            <th class="col-sm-6" colspan="2">Response</th>
+                            <th class="col-sm-1"></th>
+                        </tr>
+                        <tr>
+                            <th class="col-sm-2">Methods</th>
+                            <th class="col-sm-3">Pattern</th>
+                            <th class="col-sm-2">Method/Status</th>
+                            <th class="col-sm-4">Template</th>
+                            <th class="col-sm-1"></th>
+                        </tr>
+                    </thead>
+                    <tbody id="rules">
+                    </tbody>
+                    <tfoot style="display:none">
+                        <tr class="rule">
+                            <td>
+                                <select multiple="multiple" class="form-control requestMethod">
+                                    <option value="GET" selected="selected">GET</option>
+                                    <option value="POST">POST</option>
+                                    <option value="PUT">PUT</option>
+                                    <option value="PATCH">PATCH</option>
+                                    <option value="DELETE">DELETE</option>
+                                </select>
+                            </td>
+                            <td>
+                                <input type="text" class="form-control requestPattern" value=".*" onchange="" placeholder=".*"/>
+                            </td>
+                            <td>
+                                <select class="form-control responseType">
+                                    <!-- GET responses -->
+                                    <optgroup label="GET responses">
+                                        <option value="calli:copy">Copy (200)</option>
+                                        <option value="calli:canonical">Canonical (301)</option>
+                                        <option value="calli:alternate" selected="selected">Alternate (302)</option>
+                                        <option value="calli:describedby">Described by (303)</option>
+                                        <option value="calli:resides">Resides (307)</option>
+                                        <option value="calli:moved">Moved (308)</option>
+                                        <option value="calli:missing">Missing (404)</option>
+                                        <option value="calli:gone">Gone (410)</option>
+                                        <option value="calli:illegal">Illegal (451)</option>
+                                    </optgroup>
+                                    <!-- Other responses -->
+                                    <optgroup label="Other responses">
```

Comment:
These are not responses, but system request methods. Maybe use the label "Other requests".